modules/apps/foundation/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util.js

```diff
@@ -1,4 +1,4 @@
-;(function(A, $, _, Liferay) {
+d;(function(A, $, _, Liferay) {
```

Comment:
Make sure to look over your changes or use linters to catch accidental changes :)